### PR TITLE
add exercise to install from github/gitlab

### LINKED ISSUE #73: Add/link to version control module
R packaging is enriched substantially if the package is published on github and developed under version control. Fortunately, Rstudio makes this very easy. 
While it need not be implemented in every workshop, it is worthwhile for this lesson to contain an episode on version control, or refer to e.g. [this episode](https://carpentries-incubator.github.io/Reproducible-Publications-with-RStudio/05-setup-versioning/index.html).

### PR DESCRIPTION
**Pull request by f-hafner**
working on https://github.com/carpentries-incubator/lesson-R-packaging/issues/81
Closes #73

rewrite of existing hypothetical exercise, and combine with https://github.com/esciencecenter-digital-skills/2022-09-14-ds-rpackaging/blob/main/files/CollaborativeDocument-week1.md. also add gitlab example

<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
checkout@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

### CODE REVIEW

File: _episodes/02-accessing.md [hunk 1]
```diff
@@ -141,12 +141,18 @@ We can use GitHub to find packages or to make our own packages publicly availabl
...
 
-Let's imagine we want to install a package from GitHub, such as [github.com/PabRod/kinematics](https://github.com/PabRod/kinematics).
-The package we installed before, `devtools`, allows us to install easily using:
 
-```r
-devtools::install_github("PabRod/kinematics")
-```
+> ## Find a package on github, bitbucket, or gitlab and install it with devtools.
+> If you have trouble finding a package, try [github.com/PabRod/kinematics](https://github.com/PabRod/kinematics).
+> If you know how to install from github already, try with a package from gitlab. For instance, try [psyverse](https://gitlab.com/r-packages/psyverse)
```
Comment (bvreede) [inline _episodes/02-accessing.md:147]:
```suggestion
> If you know how to install from github already, try a package from gitlab. For instance, try [psyverse](https://gitlab.com/r-packages/psyverse)
```

File: _episodes/02-accessing.md [hunk 2]
```diff
@@ -141,12 +141,18 @@ We can use GitHub to find packages or to make our own packages publicly availabl
...
+>
+> > ## Solution
+> > ```
+    devtools::install_github("PabRod/kinematics")
+    devtools::install_git("https://gitlab.com/r-packages/psyverse")
```
Comment (bvreede) [inline _episodes/02-accessing.md:152]:
```suggestion
    devtools::install_gitlab("r-packages/psyverse")
    # or
    devtools::install_git("https://gitlab.com/r-packages/psyverse") 
```

File: _episodes/02-accessing.md [hunk 3]
```diff
@@ -141,12 +141,18 @@ We can use GitHub to find packages or to make our own packages publicly availabl
...
+    ```
+> {: .solution}
+{: .challenge}
 
 Learning how to publish your package on GitHub is out of the scope of the present course.
 But, if you are interested, we encourage you to take the course on [Version control with Git and GitHub](https://swcarpentry.github.io/git-novice/).
```
Comment (bvreede) [inline _episodes/02-accessing.md:158]: Maybe this is the point where we should mention git/github integration with Rstudio (see #73)?

**Review by bvreede — APPROVED** [review]

**Comment by bvreede** [comment]
I was quick to click "finish review", so let me add: 🎉 thanks for the work!! Looks great!

File: _episodes/02-accessing.md (hunk 3 above)
Comment (f-hafner) [reply _episodes/02-accessing.md]: good idea, fixed!

**Review by bvreede — COMMENTED** [review]
Thanks!
Two more tiny suggestions; I'm happy to merge with these!  Let me know :)

**Pull request merged**